ansible/playbook/templates/user-data.j2

```diff
@@ -0,0 +1,38 @@
+#cloud-config
+chpasswd:
+  list: |
+    root: {{ hashed_user_pw }}
+  expire: False
+
+disable_root: false
+
+package_upgrade: false
+
+yum_repos:
+  - epel:
+      baseurl: http://download.fedoraproject.org/pub/epel/7/$basearch
+      enabled: true
+      gpgcheck: false
+      name: Extra Packages for Centos 7 - $basearch
+
+packages:
+  - ansible
```

Comment:
I think that we can remove it excepted if it is needed when we execute the role remotely. WDYT ?

Reply:
I agree, we don't need ansible installed ootb.